sample/package.json

```diff
@@ -32,7 +32,7 @@
   "dependencies": {
     "@types/koa": "^2.0.48",
     "koa": "^2.7.0",
-    "oncilla": "^3.0.0",
+    "oncilla": "^4.0.0",
```

Comment:
Not related to the PR?

Reply:
It is unrelated, but hopefully ok to include vs independent PR?

Reply:
It is, I’m checking to understand whether it was intended to batch in or whether it is supposed to have a related effect on the PR.